frontend/src/component/CategoryComponent.tsx

```
@@ -0,0 +1,51 @@
+                      className="flex items-center dark:text-white text-gray-600 cursor-pointer"
+                    >
+                      <span
+                        className={`inline-block h-4 w-4 bg-${
+                          ["green", "indigo", "yellow", "blue"][index]
+                        }-300 mr-3`}
+                      ></span>
+                      {topic.name}
+                    </Link>
+                  </li>
+                )
+              )}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)